keadm/app/cmd/util/common.go

```diff
@@ -184,8 +184,8 @@ func GetOSInterface() types.OSTypeInstaller {
 	case CentOSType:
 		return &CentOS{}
 	default:
+		panic("unsupport os-release")
```

Comment:
@luguanglong , Thanks for the fix, can you re-phrase it to sound better something like "This OS version is currently un-supported by keadm"

Reply:
@naveensriram OK